ui/lib/sync/addon/components/secrets/page/destinations/destination/secrets.hbs

```diff
@@ -3,4 +3,85 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<Secrets::DestinationHeaderTabsToolbar @destination={{@destination}} />
\ No newline at end of file
+<Secrets::DestinationHeader @destination={{@destination}} />
+
+{{#if @associations.meta.filteredTotal}}
+  <div class="has-bottom-margin-s">
+    {{#each @associations as |association index|}}
+      <ListItem as |Item|>
+        <Item.content>
+          <div>
+            <span data-test-association-name={{index}}>
+              {{association.id}}
+            </span>
+            <Hds::Badge @text={{association.syncStatus}} @color="neutral" data-test-association-status={{index}} />
+            <div>
+              <code class="has-text-grey is-size-8" data-test-association-updated={{index}}>
+                last synced on
+                {{date-format association.updatedAt "MMMM do yyyy, h:mm:ss a"}}
+              </code>
+            </div>
+          </div>
+        </Item.content>
+
+        <Item.menu as |Menu|>
+          {{#if (or association.setAssociationPath.isLoading association.removeAssociationPath.isLoading)}}
+            <li class="action">
+              <LoadingDropdownOption />
+            </li>
+          {{else}}
+            <li>
+              <Hds::Button
+                @text="Sync now"
+                class="link"
+                disabled={{not association.canSync}}
+                data-test-association-action="sync"
+                {{on "click" (fn this.update association "set")}}
+              />
+            </li>
+            <li>
+              <LinkToExternal
+                class="has-text-black has-text-weight-semibold"
+                data-test-association-action="edit"
+                @route="secret"
+                @models={{array association.mount association.secretName}}
+              >
+                Edit secret
+              </LinkToExternal>
+            </li>
+            {{#if association.canUnsync}}
+              <li>
+                <Menu.Message
```

Comment:
I just replaced all of these and deleted the component 😅  - let me grab what this should be instead

Reply:
making a ticket to update after main is merged with the new `ConfirmAction`

Reply:
Perfect thanks!